WalletWasabi.Fluent/ViewModels/Login/PasswordFinder/SearchPasswordViewModel.cs

```diff
@@ -42,12 +42,13 @@ protected override void OnNavigatedTo(bool isInHistory, CompositeDisposable disp
 
 		var t = Task.Run(() => FindPassword(Options, cancelToken.Token));
 
-		Disposable.Create(async () =>
+		disposables.Add(Disposable.Create(async () =>
 		{
 			cancelToken.Cancel();
 			await t;
-		})
-		.DisposeWith(disposables);
+		}));
+
+		disposables.Add(cancelToken);
```

Comment:
This is confusing really. `cancelToken` should be called `cancellationTokenSource` / `cts` / ...

Reply:
#7021